cmd/acra-keys/keys/list-keys.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020, Cossack Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package keys
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/cossacklabs/acra/keystore"
+	"github.com/cossacklabs/acra/utils"
+)
+
+// PrintKeys prints key list prettily into the given writer.
+func PrintKeys(keys []keystore.KeyDescription, writer io.Writer, params *CommandLineParams) error {
+	if params.UseJSON {
+		return printKeysJSON(keys, writer)
```

Comment:
can we add test which will parse keys printed in json format? text format maybe hard to parse but json we can check and validate that it really machine readable.

Reply:
> can we add test which will parse keys printed in json format?

I've added a couple of tests for `PrintKeys`. For default format it simply verifies that the output is the expected one. For JSON we try to parse the output as JSON and check it successfully parses into the same data as the input.